src/store/middlewares/settings.js

```diff
@@ -0,0 +1,21 @@
+import actionsType from '../../constants/actions';
+import { settingsUpdated } from '../../actions/settings';
+import { pricesRetrieved } from '../../actions/service';
+
+const settings = store => next => (action) => {
+  switch (action.type) {
+    case actionsType.settingsUpdateToken:
+      next(action);
+      // istanbul ignore next
```

Comment:
Why ignore? Please add a comment to the file

Reply:
I just fixed the file now it is testing properly, this is something that I forget to fix thx